### PR TITLE
update docker scripts

### PR DESCRIPTION
**Pull request by TheOnlyWei**
update docker scripts

### CODE REVIEW

File: docker/scripts/Test-AzsPowershell.ps1 [hunk 1]
```diff
@@ -8,11 +8,13 @@
 param (
     [Parameter(HelpMessage = "The object ID associated with the secret or certificate AzureStack login information. Admin tests will fail if invalid.")]
     [string]
-    $ClientObjectId = $ENV:ClientObjectId
+    $ClientObjectId = $ENV:ClientObjectId,
+    [Parameter(HelpMessage = "The location of the resources for Azure Stack.")]
+    [string]
+    $resourceLocation = $ENV:Location
```
Comment (bganapa) [inline docker/scripts/Test-AzsPowershell.ps1:14]: do we have a readme where we need to update this new parameter?

**Review by sarathys — APPROVED** [review]

**Pull request merged**